delayed_job.gemspec

```diff
@@ -13,3 +14,4 @@ Gem::Specification.new do |spec|
   spec.test_files     = Dir.glob('spec/**/*')
   spec.version        = '4.0.6'
 end
+# rubocop:enable SpaceAroundOperators
```

Comment:
I think the disable comments are scoped by file, so re-enabling isn't needed in this case, because it will be enabled for other files [anyway][0]... but it doesn't hurt! :sweat_smile: 

[0]: https://cloud.githubusercontent.com/assets/1421211/9827935/79e47b54-58b5-11e5-9cc8-fe30a9ffd017.png